src/transformers/models/blenderbot/modeling_tf_blenderbot.py

```diff
@@ -1478,9 +1478,15 @@ def _reorder_cache(past, beam_idx):
             )
         return (past[0], reordered_past)
 
-    def adjust_logits_during_generation(self, logits, cur_len, max_length):
-        if cur_len == max_length - 1:
-            vocab_range = tf.constant(range(self.config.vocab_size))
-            return tf.where(vocab_range != self.config.eos_token_id, LARGE_NEGATIVE, logits)
-        else:
-            return logits
+    # Copied from transformers.models.bart.modeling_tf_bart.TFBartForConditionalGeneration.compute_loss
```

Comment:
I think this was a merge conflict here. I deleted this function a while back and added the more generic `TFCausalLanguageModelingLoss` to all BART-like models. Can you check whith current master and delete all those newly added `compute_loss` functions?